DTS-STN/canadian-dental-care-plan
# Removing unused address and user services and mocks

### Description
The use of these services has since been replaced by `personal-information-service`.

### Checklist

### Test Instructions
Run the application and navigate to `/en/personal-information`. The page should display as before.
